modules/RTCStats/RTCStats.ts

```diff
@@ -0,0 +1,189 @@
+import rtcstatsInit from '@jitsi/rtcstats/rtcstats';
+import traceInit from '@jitsi/rtcstats/trace-ws';
+import EventEmitter from 'events';
+
+import { 
+    CONFERENCE_JOINED, 
+    CONFERENCE_LEFT, 
+    CONFERENCE_UNIQUE_ID_SET 
+} from '../../JitsiConferenceEvents';
+import JitsiConference from '../../JitsiConference';
+
+const RTC_STATS_WC_DISCONNECTED = 'rtcstats_ws_disconnected';
```

Comment:
These need to be exported so they can be attached in the JitsiMeetJS.events.rtcstats object.

Reply:
I've added separate events and bubbled them up, using the template from other services. Adding the changes to jitsi-meet asap.